concent_api/core/tests/test_integration_force_payment.py

```diff
@@ -330,41 +340,58 @@ def test_provider_send_correct_force_payment_concent_should_accept(self):
+                payment_ts      = "2018-02-05 11:55:00",
                 task_to_compute = self._get_deserialized_task_to_compute(
                     timestamp                       = "2018-02-05 10:00:00",
                     deadline                        = "2018-02-05 10:00:10",
                     task_id                         = '2',
-                    requestor_ethereum_public_key   = self._get_encoded_requestor_ethereum_public_key(),
+                    requestor_public_key            = self.REQUESTOR_PUBLIC_KEY,
+                    requestor_ethereum_public_key   = self._get_requestor_ethereum_public_key(),
+                    provider_public_key             = self.PROVIDER_PUBLIC_KEY,
+                    provider_ethereum_public_key    = self._get_provider_ethereum_public_key(),
+                    price                           = 15000,
                 )
             ),
             self._get_deserialized_subtask_results_accepted(
                 timestamp       = "2018-02-05 9:00:15",
-                payment_ts      = "2018-02-05 11:00:00",
+                payment_ts      = "2018-02-05 11:55:00",
                 task_to_compute = self._get_deserialized_task_to_compute(
                     timestamp                       = "2018-02-05 9:00:00",
                     deadline                        = "2018-02-05 9:00:10",
                     task_id                         = '3',
-                    requestor_ethereum_public_key   = self._get_encoded_requestor_ethereum_public_key(),
+                    requestor_public_key            = self.REQUESTOR_PUBLIC_KEY,
+                    requestor_ethereum_public_key   = self._get_requestor_ethereum_public_key(),
+                    provider_public_key             = self.PROVIDER_PUBLIC_KEY,
+                    provider_ethereum_public_key    = self._get_provider_ethereum_public_key(),
+                    price                           = 7000,
                 )
             )
         ]
+
         serialized_force_payment = self._get_serialized_force_payment(
             timestamp                     = "2018-02-05 12:00:20",
             subtask_results_accepted_list = subtask_results_accepted_list
         )
 
         with freeze_time("2018-02-05 12:00:20"):
-            with mock.patch('core.message_handlers.base.get_list_of_transactions', _get_requestor_valid_list_of_transactions):
-                with mock.patch('core.message_handlers.base.payment_summary', _get_payment_summary_value_100):
-                    with mock.patch('core.message_handlers.base.make_payment_to_provider', _make_payment_to_provider):
+            with mock.patch('core.message_handlers.base.get_number_of_eth_block', self._get_number_of_eth_block):
+                with mock.patch('core.message_handlers.base.make_force_payment_to_provider', self._make_force_payment_to_provider):
+                    fake_responses = [mock.Mock(), mock.Mock()]
```

Comment:
I think these two shouldn't be created, since you are overriding them in lines below

Reply:
Done